.demo/env/clouds/aws/demo/jobs.demo.yaml

```diff
@@ -0,0 +1,28 @@
+teamConfig:
+    teams:
+        demo:
+            jobs:
+                - deployAlways: true
```

Comment:
Maurice, it seems some clarification is needed.
In case of teams job how does a user define "things" to calculate hash from.  And what are "things"?
Please be more specific.

Reply:
Like I said, please look at that chart

Reply:
Err, why not stick with simple if else logic and check for equality for the presence of this key?

Reply:
Computing hash is needed for implementing deployment onChange.
The onChange flag means that the job will be redeployed only if any ConfigMap/Secret that is referenced in Job/CronJob changes.

However, the goal of this PR has been reached - teams can define jobs/cronJobs. We just wait for final tests.

The onChange functionality is something else and should NOT be part of this PR. 
This PR is already too big and there is too much feedback that is hard to maintain and makes reviewer and author overwhelmed.

Reply:
Alright, thanks again.

Reply:
I split this in #287.